### PR TITLE
Fix links to orcharhino docs, add image, and minor improvements

### PR DESCRIPTION
**Pull request by maximiliankolb**
I suggest you review this commit by commit.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.3/Katello 4.5
* [x] Foreman 3.2/Katello 4.4
* [x] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/common/modules/con_introduction-to-application-life-cycles.adoc [hunk 1]
```diff
@@ -31,10 +31,14 @@ When the test server passes the initial phase, you can set the production-level
...
+ifdef::foreman-el,foreman-deb,katello[]
 image::application_life_cycle.png[title="The {Project} Application Life Cycle", alt="The {Project} Application Life Cycle"]
 endif::[]
 
 ifdef::satellite[]
 image::application_life_cycle_sat.png[title="The {Project} Application Life Cycle", alt="The {Project} Application Life Cycle"]
 endif::[]
+
+ifdef::orcharhino[]
+image::common/application_life_cycle_orcharhino.svg[title="The {Project} Application Life Cycle", alt="The {Project} Application Life Cycle"]
```
Comment (ekohl) [inline guides/common/modules/con_introduction-to-application-life-cycles.adoc:43]: How does this differ from the existing PNG, other than the file format? Can't we use this image for both katello and orcharhino?

File: guides/common/modules/con_introduction-to-application-life-cycles.adoc [hunk 2]
```diff
@@ -31,10 +31,14 @@ When the test server passes the initial phase, you can set the production-level
 Another example is a development life cycle for a software product.
 To develop a new piece of software in a development environment, test it in a quality assurance environment, pre-release as a beta, then release the software as a production-level application.
 
-ifndef::satellite[]
+ifdef::foreman-el,foreman-deb,katello[]
```
[the remark this answers is not in the document]
Comment (maximiliankolb) [reply guides/common/modules/con_introduction-to-application-life-cycles.adoc:34]: Applied!

File: guides/common/modules/con_introduction-to-application-life-cycles.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/con_introduction-to-application-life-cycles.adoc:43]: Both files use slightly different shades of blue: one for orcharhino (acccording to our internal style guide), one I got from theforeman.org.

**Comment by ekohl** [comment]
> The following output files are affected by this PR:

Unrelated, but I think this clearly indicates we have a lot of duplication because of the common/images directory. Probably not that big of a deal that we need to solve now, but it's fairly inefficient.

File: guides/common/modules/con_introduction-to-application-life-cycles.adoc (hunk 1 above)
Comment (ekohl) [reply guides/common/modules/con_introduction-to-application-life-cycles.adoc:43]: To me the shades of blue are not that important and less duplication is less maintenance work and thus better. I'll merge it now, but perhaps @Lennonka can weigh in on this.

**Comment by ekohl** [comment]
To github.com:theforeman/foreman-documentation
   ed2b91bbb..24d8e109b  3.1 -> 3.1
   f882fd084..f1f9d7e18  3.2 -> 3.2
   774d7499f..9954f2228  3.3 -> 3.3

**Pull request merged**